geometry/eolearn/geometry/transformations.py

```diff
@@ -207,7 +207,7 @@ def _vector_data_to_shape_iterator(self, vector_data, join_per_value):
         if join_per_value:
             classes = np.unique(vector_data[self.values_column])
             grouped = (vector_data.geometry[vector_data[self.values_column] == cl] for cl in classes)
-            grouped = (shapely.ops.cascaded_union(group) for group in grouped)
+            grouped = (shapely.ops.unary_union(group) for group in grouped)
```

Comment:
Should we then add `shapely>=1.8.0` into requirements (this is a few months old version), or should we perhaps here do something like we have in [here in sh-py](https://github.com/sentinel-hub/sentinelhub-py/blob/v3.4.3/sentinelhub/areas.py#L105)?

Reply:
Not a big fan of checking versions at runtime, but also not a big fan of requirements restrictions. We can use the same trick yeah.